docs/src/features.md

```diff
@@ -1,33 +1,18 @@
 
 ## Commands and shortcuts
 
-### Plugin commands
+Apart from the commands listed in the [Keyboard Shortcuts](keyboard_shortcuts.md) section, LSP also provides more generic commands in the Command Palette:
```

Comment:
```suggestion
LSP provides the following commands in the Command Palette:
```

The beginning of the sentence is maybe unnecessary, because you also mention the keyboard shorcuts in the bullet point bellow:
`Preferences: LSP Key Bindings`: opens LSP key bindings configuration - see [Keyboard Shortcuts](keyboard_shortcuts.md)`

Reply:
Yes but the point was that those listed below are not the only ones that LSP provides.
But I don't have very strong feeling about that either way.

Reply:
I also do not have a preference here, it just felt a bit redundant.